client/components/mma/switch/SwitchReview.tsx

```diff
@@ -51,6 +51,19 @@ const SwitchReview = () => {
 		<>
 			<section css={sectionSpacing}>
 				<Stack space={3}>
+					<section>
+						<Heading sansSerif>Review change</Heading>
+						<p
+							css={css`
+								${textSans.medium()};
+							`}
+						>
+							You will now support us with {mainPlan.currency}
+							{threshold} every month, giving you exclusive
```

Comment:
I guess we want to make 'month' dynamic as well, although I hardcoded it myself in the section below this 🙃  We could tackle that separately once we have all the content on the page though.

Reply:
Good spot, I've made the change now since it's easy enough